examples/PySDM_examples/Berry_1967/example.py

```diff
@@ -15,7 +15,7 @@
 
 def main(plot: bool, save):
     with np.errstate(all="ignore"):
-        u_term_approxs = (gunn_and_kinzer.Interpolation,)
+        u_term_approxs = (gunn_and_kinzer.GunnKinzer1949,)
```

Comment:
should these now be strings passed to Formulae ctor?

Reply:
applied